content/sensu-go/6.7/observability-pipeline/observe-schedule/checks.md

```diff
@@ -1318,6 +1519,167 @@ ttl: 100
 {{< /code >}}
 {{< /language-toggle >}}
 
+#### Check output truncation attributes
+
+|max_output_size  | |
+-------------|-------
+description  | Maximum size of stored check outputs. In bytes. When set to a non-zero value, the Sensu backend truncates check outputs larger than this value before storing to etcd. `max_output_size` does not affect data sent to Sensu filters, mutators, and handlers.
+required     | false
+type         | Integer
+example      | {{< language-toggle >}}
+{{< code yml >}}
+max_output_size: 1024
+{{< /code >}}
+{{< code json >}}
+{
+  "max_output_size": 1024
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+|discard_output  | |
+-------------|------
+description  | If `true`, discard check output after extracting metrics. No check output will be sent to the Sensu backend. Otherwise, `false`.
+required     | false
+type         | Boolean
+example      | {{< language-toggle >}}
+{{< code yml >}}
+discard_output: true
+{{< /code >}}
+{{< code json >}}
+{
+  "discard_output": true
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+#### `output_metric_tags` attributes
+
+name         | 
+-------------|------
+description  | Name for the [output metric tag][19].
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+name: instance
+{{< /code >}}
+{{< code json >}}
+{
+  "name": "instance"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+value        | 
+-------------|------
+description  | Value for the [output metric tag][19]. Use [check token substitution][39] syntax for the `value` attribute, with dot-notation access to any event attribute.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+value: {{ .name }}
+{{< /code >}}
+{{< code json >}}
+{
+  "value": "{{ .name }}"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+#### `output_metric_thresholds` attributes
+
+name         | 
+-------------|------
+description  | Name of the metric to use for [metric threshold evaluation][79]. Must match the [event.metrics.points[].name][81] value for a metric point in the check results.
```

Comment:
Note that `output_metric_format` is required to make sure that metrics are present. Metric thresholds are only evaluated if that value is set.